### PR TITLE
Update to latest range message, and a few tweaks

### PR DESCRIPTION
**Pull request by slimdave**
I just tweaked a rake method name, some yaml-file logic, and updated to the latest version of the range message

### CODE REVIEW

File: lib/lisbn/ranges.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ def self.to_yaml
       ranges = xml["ISBNRangeMessage"]["RegistrationGroups"]["Group"]
 
       hash = Array(ranges).flatten.inject({}) do |memo, group|
-        prefix = group["Prefix"].gsub(/-/, '')
+        prefix = group["Prefix"].gsub(/-/, '').to_s
```
Comment (ragalie) [inline lib/lisbn/ranges.rb]: Why is this necessary? I'm surprised something implements `gsub` but doesn't return a string.

File: tasks/save_ranges_to_json.rake [hunk 2]
```diff
@@ -3,5 +3,5 @@ require "nori"
 require_relative "../lib/lisbn/ranges"
 
 task :save_ranges_to_json do
-  Lisbn::Ranges.save_to_json!
+  Lisbn::Ranges.save_to_yaml!
```
Comment (ragalie) [inline tasks/save_ranges_to_json.rake]: Might be a good idea to update the task and file names too while you're doing this.

File: lib/lisbn/ranges.rb (hunk 1 above)
Comment (slimdave) [reply lib/lisbn/ranges.rb]: Yes, I was also surprised by this, but without it the range.yml ends up with:
```yaml
  :length: 7
!ruby/string:Nori::StringWithAttributes '9781':
- :range: !ruby/range
```

With the to_s in place we get:

```yaml
  :length: 7
'9781':
- :range: !ruby/range
```

I've not looked into the details of why, though.

File: tasks/save_ranges_to_json.rake (hunk 2 above)
Comment (slimdave) [reply tasks/save_ranges_to_json.rake]: Good idea. Will do that.

**Comment by ragalie** [comment]
Thanks! Pushed a new version of the gem (0.2.3) with your changes.

**Comment by slimdave** [comment]
Cheers!
